cblox/include/cblox/core/submap_collection_inl.h

```diff
@@ -75,6 +75,18 @@ SubmapID SubmapCollection<SubmapType>::createNewSubMap(
   return new_ID;
 }
 
+template <typename SubmapType>
+void SubmapCollection<SubmapType>::createNewSubMap(
+    const Transformation &T_G_S, const SubmapID submap_id,
+    voxblox::TsdfMap::Ptr tsdf_map_ptr) {
```

Comment:
The implementation here seems reasonable, however, it seems weird that we would have a TsdfMap not associated with a submap... Where is this occurring?